docs/docs/troubleshooting/building-issues.md

```diff
@@ -29,6 +29,8 @@ A `devicetree error` followed by a reference to the line number on `<keyboard>.k
 devicetree error: /__w/zmk-config/zmk-config/config/cradio.keymap:109 (column 4): parse error: expected ';' or ','
 ```
 
+Note that the provided number for the column corresponds to the numbering after the preprocessor expansion and it will likely not match the column numbers in your keymap.
```

Comment:
```suggestion
:::warning
The provided number for the column corresponds to the numbering after the preprocessor expansion and will likely not match the column numbers in your keymap.
:::
```
Warning block, maybe?

Reply:
Hmmm, a warning block doesn't feel quite right on re-read. I would like something to break apart the different devicetree compilation errors, though. Any ideas?

Reply:
Yeah, I didn't think it was worth attracting any soecial attention to. For different errors, we can do subsections or bullet points?

Reply:
Would be some massive bullet points. Perhaps subsections if you can think of nice names, or maybe hrules?

Reply:
I didn't quite like the hrules as a distinguisher: 
![image](https://github.com/user-attachments/assets/3446bee2-c98d-419b-a48d-9991312daaeb)

Pushing a version with subsection headers now.